modules/docs/mdx/11.0-UPGRADE-GUIDE.mdx

```diff
@@ -15,10 +18,11 @@ any questions.
   - [Text Area Preview](#text-input-preview)
   - [Label Text](#label-text)
 - [Component Updates](#component-updates)
+  - [Style Updates](#style-updates)
   - [Card](#card)
   - [Count Badge](#count-badge)
   - [Text](#text)
-  - [Form Field Preview](#form-field-preview)
+	- [FormField Preview](#form-field-preview)
```

Comment:
this seems off? also add the space back in

Reply:
Agreed, addressed.